_test/test_sqw/test_PixelData_operations.m

```diff
@@ -4,195 +4,14 @@
         BYTES_PER_PIX = PixelData.DATA_POINT_SIZE*PixelData.DEFAULT_NUM_PIX_FIELDS;
         SIGNAL_IDX = 8;
         VARIANCE_IDX = 9;
-        
         ALL_IN_MEM_PG_SIZE = 1e12;
         FLOAT_TOLERANCE = 4.75e-4;
-        
-        this_dir = fileparts(mfilename('fullpath'));
-        test_sqw_file_path = '../test_sqw_file/sqw_1d_1.sqw';
-        test_sqw_2d_file_path = '../test_sqw_file/sqw_2d_1.sqw';
-        ref_npix_data = [];
-        ref_s_data = [];
-        ref_e_data = [];
-        
-        pix_in_memory_base;
-        pix_in_memory;
-        pix_with_pages_base;
-        pix_with_pages;
-        
-        pix_with_pages_2d;
-        ref_npix_data_2d;
-        ref_s_data_2d;
-        ref_e_data_2d;
-        
-        old_warn_state;
     end
     
     methods
         
         function obj = test_PixelData_operations(~)
             obj = obj@TestCase('test_PixelData_operations');
-            
-            addpath(fullfile(obj.this_dir, 'utils'));
```

Comment:
Rather than cause a surprise if anyone looks at the file history, it may be worth saying where the tests were moved to.

Reply:
I feel as though if someone's looking at the file history, then the commit message ("_Refactor compute_bin_data tests into own file_"), and the full commit diff, will make clear what's happened.

Reply:
I would note that reviewing any PR through the file changes does not give the commit message history. It should be easy to review without having to second-guess what the developer did and go looking for other access points to the reasoning.

Reply:
I'd argue this is the job of the PR description, not a comment within code. If the purpose of a comment is to guide a reviewer, it becomes redundant as soon as the PR is merged.